### DISCUSSION TITLE
Why Line Width does not automatically change when selecting a printer with a different nozzle size?

### DISCUSSION THREAD

**Discussion by deemoss** [OrcaSlicer/OrcaSlicer#2304] [General]
I checked the printer settings, and their profiles have different size nozzles. However, when I select a printer with a different nozzle size the Line Widths do not change. If I set Line Widths to 100%, the widths are calculated based on the correct nozzle size...

Am I doing something wrong?

<img width="1107" alt="1" src="https://github.com/SoftFever/OrcaSlicer/assets/2356886/4107c0e5-e302-4270-9ed5-cf186750b530">
<img width="1163" alt="2" src="https://github.com/SoftFever/OrcaSlicer/assets/2356886/9e110a0b-fcc0-4f4c-a6c5-1ea88a45ecaa">

**Comment by darco415** [comment]
I just sent my line width to 120% and swap out nozzles..not sure why Orca doesnt update when choosing mm.

**Comment by KingBain** [comment]
Just ran into this exact issue,  Coming from a cura slicer world, this surprised me and is forcing me to redo all my calibrations and default print settings
